workshop/Lab0/README.md

```diff
@@ -0,0 +1,114 @@
+# Lab 0: Install Prequisites
```

Comment:
maybe link to kube101 in this intro so they have a starting point.

Reply:
I'm not sure what the entry point is for people, so this may need to be in the toplevel README.